packages/passport/sdk/src/guardian/index.ts

```diff
@@ -228,13 +235,16 @@ export default class GuardianClient {
     }
   }
 
-  private async evaluateMessage(
-    { chainID, payload }:GuardianMessageValidationParams,
+  private async handleEIP712MessageEvaluation(
+    { chainID, payload }:GuardianEIP712MessageEvaluationParams,
   ): Promise<guardian.MessageEvaluationResponse> {
     try {
       const user = await this.authManager.getUserZkEvm();
       if (user === null) {
-        throw new PassportError('evaluateMessage requires a valid ID token or refresh token. Please log in first', PassportErrorType.NOT_LOGGED_IN_ERROR);
+        throw new PassportError(
+          'User not logged in. Please log in first.',
+          PassportErrorType.NOT_LOGGED_IN_ERROR,
+        );
       }
```

Comment:
Can we throw a `JsonRpcError` error here please? The `PassportError` were introduced with starkEx, but the `JsonRpcError` adheres to EIP-1193. `ProviderErrorCode.UNAUTHORIZED` is the closest error code that we have.